deepchem/metrics/tests/test_normalize.py

```diff
@@ -0,0 +1,111 @@
+"""Test normalization of input."""
+
+import numpy as np
+import unittest
+import deepchem as dc
+from deepchem.metrics import to_one_hot
+from deepchem.metrics import from_one_hot
+from deepchem.metrics import normalize_prediction_shape
+from deepchem.metrics import normalize_weight_shape
+
+class TestNormalization(unittest.TestCase):
+  """
+  Tests that input normalization works as expected.
+  """
+
+  def test_one_hot(self):
+    """Test the one hot encoding."""
+    y = np.array([0, 0, 1, 0, 1, 1, 0])
+    y_hot = to_one_hot(y)
+    expected = np.array([[1, 0], [1, 0], [0, 1], [1, 0], [0, 1], [0, 1], [1,
+                                                                          0]])
+    yp = from_one_hot(y_hot)
+    assert np.array_equal(expected, y_hot)
+    assert np.array_equal(y, yp)
+
+  def test_normalize_scalar_classification_binary(self):
+    """Tests 1d classification normalization."""
+    y = 1 
+    y_out = normalize_prediction_shape(y, mode="classification")
+    assert y_out.shape == (1, 1, 2)
```

Comment:
In all the tests in this file, we should check that the output actually has the correct value, not just the correct shape.

Reply:
Good suggestion! Adding in tests for correct values